src/features/gameboard/events/event-definitions/tasks/cleanMess.ts

```diff
@@ -1,25 +1,38 @@
-import { GameEvent } from '../../../types'
+import { GameEvent, PlayerParts } from '../../../types'
 import { GameBoardActions } from '../../../store'
 import { MessageType } from '../../../MessageArea/MessageTypes'
 
-export const lickPre: GameEvent = () => {
+export const cleanMess: GameEvent = () => {
   return async (state, dispatch) => {
+    let msg
+    switch (state.settings.player.parts) {
+      case PlayerParts.Cock:
+        msg = "pre you've leaked"
+        break
+      case PlayerParts.Pussy:
+        msg = "juices you've made"
+        break
+      case PlayerParts.Neuter:
+      default:
+        msg = "mess you've made"
+        break
```

Comment:
Nit pick: I could see these also being formatMessage strings, maybe `$fluid-affirmation`? Just to have them all in one place